polysemy-plugin/test/ExampleSpec.hs

```diff
@@ -33,7 +34,13 @@ program = catch @CustomException work $ \e -> writeTTY ("Caught " ++ show e)
             _             -> writeTTY i >> writeTTY "no exceptions"
 
 foo :: IO (Either CustomException ())
-foo = (runM .@ lowerResource .@@ lowerError @CustomException) $ teletypeToIO program
+foo =
+    runFinal
+  . embedToFinal @IO
+  . resourceToIOFinal
+  . errorToIOFinal @CustomException
+  . teletypeToIO
+  $ program
```

Comment:
This stings. The tests for `polysemy-plugin` use the now deprecated `lower-` interpreters, so they'd generate warnings without this, but this also requires a dependency bump for polysemy-plugin from `polysemy>=0.6` to `polysemy>=1.2`.

I'm not sure what the best solution to this is.

Reply:
Oh yeah, there are separate dependencies for the tests. Then there's no issue.